_playwright-tests/tests/API/Templates.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { expect } from '@playwright/test';
+import {test, expectError} from "./base_client";
+import {FeaturesApi, PopularRepositoriesApi, ResponseError, TemplatesApi} from "./client";
```

Comment:
Do you need `PopularRepositoriesApi` in this test?

Reply:
nope! cleaned up some imports